package.json

```diff
@@ -3,6 +3,7 @@
   "dependencies": {
     "@koa/cors": "^3.0.0",
     "@koa/router": "^8.0.5",
+    "async-iterator-all": "^1.0.0",
```

Comment:
Useful to add types for this. Fancy having a crack at it?

In https://github.com/libero/article-store/blob/60161cffe176be651ef74af5af02c56d0c3a19ab/src/routes/article-list.ts#L11 `count` is correctly a number, but `list` is `any`.

Reply:
(Though maybe there is already a typed alternative.)

Reply:
Can't see one; unclear, but looking like the library has been replaced by https://www.npmjs.com/package/it-all (same author).

Reply:
> Fancy having a crack at it?

I will have a go at creating a PR for https://github.com/DefinitelyTyped/DefinitelyTyped

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41119